docs/Home.md

```diff
@@ -0,0 +1,88 @@
+1. Enter your project name then select your JSON key that was created in the preceding steps.
+1. Click OK.
+
+### Enable Cloud Resource Manager API
+
+1. Go to the [Cloud Resource Manager API](https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview) overview in the GCP console.
+1. Make sure that your current project is the same used above.
+1. Click Enable. This will redirect you to the API & Services dashboard for Cloud Resource Manager.
+1. Click credentials to ensure that your jenkins-gke service account credentials are compatible.
+    - Note: The necessary permissions are included in both the iam.serviceAccountUser and compute.networkViewer roles which were added above.
+
+### Configure GKE Cluster
+
+1. Create GKE cluster (if not already exists).
+    ```bash
+    export CLUSTER=my-jenkins-cluster
+    export ZONE=us-central1-c
+    gcloud container clusters create --zone $ZONE $CLUSTER
+    ```
+1. Enable legacy authorization on the cluster.
+    ```bash
+    gcloud container clusters update --enable-legacy-authorization --zone $ZONE $CLUSTER
+    ```
+1. Get credentials for the cluster.
+    ```bash
+    gcloud container clusters get-credentials --zone $ZONE $CLUSTER
+    ```
+
+### Configure Kubernetes Cluster Permissions
+
+1. Add the cluster-admin role to the service account group within your Kubernetes cluster:
+    ```bash
+    kubectl create clusterrolebinding serviceaccounts-cluster-admin --group=system:serviceaccounts --clusterrole=cluster-admin
+    ```
+
+### Google Kubernetes Engine Builder configuration
+
+Each GKE Builder configuration can point to a different GKE cluster. Follow the steps below to create one.
+
+1. Go to Jenkins home, and select the project to be published to GKE.
+1. Click the "Configure" button on the left nav-bar.
+1. At the bottom of the page there will be a button labeled "Add build step", click the button then click Google Kubernetes Engine.
+1. In the Service Account Credentials dropdown, select the credentials that you uploaded earlier.
+1. Select the Project ID housing the GKE cluster to be published to.
+1. Select the Zone housing the GKE cluster to be published to.
+1. Select the Cluster to be published to.
+1. Enter the file path of the Kubernetes [manifest](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) within your project to be used for deployment.
+
+### Jenkins environment configuration
+
+<!--- TODO(stephenshank): Link to an image that adds kubectl to the existing jenkins agent image: https://hub.docker.com/r/jenkinsci/jnlp-slave/ --->
+
+The GKE Jenkins plugin requires the [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) binary to be installed within the Jenkins agent environment.
+
+To install a custom version of the GKE plugin in the Jenkins master environment, see the [snapshot installation instructions](SnapshotInstallation.md) for instructions on how to build and install the plugin from source.
```

Comment:
I think this line can be removed, as the section it mentioned directly follows.

Reply:
Done. Just to clarify though, are these wiki pages going to be concatenated together, and if so in any particular order? Would it make sense to move some other things on this main page to separate documents?

Reply:
I think for now there isn't enough content to merit breaking out. But we can play it by ear going forward.